Core/Core/API/APICourse.swift

```diff
@@ -66,7 +66,7 @@ public struct APICourse: Codable, Equatable {
     // let blueprint: Bool?
     // let blueprint_restrictions: ?
     // let blueprint_restrictions_by_object_type: ?
-    let image_download_url: URL? // include[]=course_image
+    let image_download_url: String? // include[]=course_image
```

Comment:
Can you add a comment here about this sometimes being an empty string so people know not to switch back to URL in the future.